manifests/init.pp

```diff
@@ -152,4 +171,50 @@
   if $mailman {
     include ::postfix::mailman
   }
-}
+
+  include postfix::configs_
+  include postfix::hashs_
+  Class['postfix::files'] -> Class['postfix::configs_']
+  Class['postfix::files'] -> Class['postfix::hashs_']
+
+}  # class postfix
+
+
+###################################################
+##
+##  Please do not include the below classes directly in the nodes
+##
+###################################################
+
+
+class postfix::configs_ {
```

Comment:
This is not a standard way of organizing Puppet code. If you want to make private classes, you can use the `assert_private()` function from stdlib.